etc/CA/certs/04.pem

```diff
@@ -0,0 +1,99 @@
+Certificate:
```

Comment:
No certs or keys should be included in the repo. Its like exposing passwords. These certs we will expect to exist already on the Raspberry Pi so we only need to reference them by location we dont have to include them.

Reply:
Understood